android-components/components/browser/thumbnails/src/test/java/mozilla/components/browser/thumbnails/utils/ThumbnailDiskCacheTest.kt

```diff
@@ -43,7 +44,32 @@ class ThumbnailDiskCacheTest {
             true
         }
 
-        cache.putThumbnailBitmap(testContext, request.id, bitmap)
+        cache.putThumbnailBitmap(testContext, ImageSaveRequest(request.id, request.isPrivate), bitmap)
+
+        val data = cache.getThumbnailData(testContext, request)
+        assertNotNull(data!!)
+        Assert.assertEquals("Hello World", String(data))
+    }
+
+    @Test
+    fun `Writing and reading bitmap bytes for private cache for sdk higher than 29`() {
```

Comment:
```suggestion
    fun `Writing and reading bitmap bytes for private cache`() {
```

This test has nothing to do with SDK levels unlike the other two.